src/Clinics/Domain/Models/Clinic.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Lightit\Clinics\Domain\Models;
+
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\Relations\BelongsToMany;
+use Lightit\Appointments\Domain\Models\Appointment;
+use Lightit\Doctors\Domain\Models\Doctor;
+
+/**
+ * @property int                          $id
+ * @property \Carbon\CarbonImmutable|null $created_at
+ * @property \Carbon\CarbonImmutable|null $updated_at
+ *
+ * @method static \Illuminate\Database\Eloquent\Builder<static>|Clinic newModelQuery()
+ * @method static \Illuminate\Database\Eloquent\Builder<static>|Clinic newQuery()
+ * @method static \Illuminate\Database\Eloquent\Builder<static>|Clinic query()
+ * @method static \Illuminate\Database\Eloquent\Builder<static>|Clinic whereCreatedAt($value)
+ * @method static \Illuminate\Database\Eloquent\Builder<static>|Clinic whereId($value)
+ * @method static \Illuminate\Database\Eloquent\Builder<static>|Clinic whereUpdatedAt($value)
+ *
+ * @property string $name
+ * @property string $address
+ *
+ * @method static \Illuminate\Database\Eloquent\Builder<static>|Clinic whereAddress($value)
+ * @method static \Illuminate\Database\Eloquent\Builder<static>|Clinic whereName($value)
+ *
+ * @property-read \Illuminate\Database\Eloquent\Collection<int, Doctor> $doctors
+ * @property-read int|null $doctors_count
+ * @property string $assigned_at
+ * @property string $status
+ * @property-read \Illuminate\Database\Eloquent\Collection<int, Doctor> $doctor
+ * @property-read int|null $doctor_count
+ *
+ * @method static \Illuminate\Database\Eloquent\Builder<static>|Clinic whereAssignedAt($value)
+ * @method static \Illuminate\Database\Eloquent\Builder<static>|Clinic whereStatus($value)
+ *
+ * @mixin \Eloquent
+ */
+class Clinic extends Model
```

Comment:
Consider grouping the annotation `@property`s together for improved readability. That would also improve the quality of your code. The same goes for `@property-read`s, `@method`s, etc.